index.js

```diff
@@ -0,0 +1,29 @@
+require('./src/yaml')
+const { LoggerFactory } = require('logger.js')
+const logger = LoggerFactory.getLogger('main', 'cyan')
+const config = require('./config.yml')
+const languages = {
+  en: require('./lang/en.json'),
+}
+const data = require('./src/data')
+const dispatcher = require('bot-framework/dispatcher')
+const Discord = require('discord.js')
+const client = new Discord.Client()
+
+client.on('ready', () => {
+  client.user.setActivity(`${config.prefix}help | ${client.guilds.size} guilds`)
+  logger.info('Bot is ready!')
+})
+
+client.on('message', async msg => {
+  const guild = await data.getServer(msg.guild.id).then(g => g.toJSON())
+  const user = await data.getUser(msg.author.id).then(u => u.toJSON())
```

Comment:
Is there a specific reason to use toJSON?
AFAIK, it provides getters so you can access data directly without converting to plain object.

Reply:
I didn't know this at that moment but I know this now, so I'll fix this.